pyproject.toml

```diff
@@ -35,6 +35,7 @@ qualang-tools = { version = "==0.14.0", optional = true }
 setuptools = { version = ">67.0.0", optional = true }
 laboneq = { version = "==2.21.0", optional = true }
 qibosoq = { version = ">=0.0.4,<0.2", optional = true }
+matplotlib = "^3.8.2"
```

Comment:
Please, move this in the section `tool.poetry.group.docs.dependencies` down below (these are dependencies not required at runtime by `qibolab`, but used to compile the documentation, including the examples)